packages/slice-machine/lib/models/common/widgets/ctBuilderArray.ts

```diff
@@ -3,6 +3,7 @@ import * as Widgets from "./withGroup";
 export default [
   Widgets.UID,
   Widgets.Group,
+  Widgets.RepeatableGroup,
```

Comment:
✍️ potential change

I think I would put it before UID, or at least ensure it's on the same line? WDYT, maybe it's something we should ask design. I feel hard to see that there are two groups now.

![Screenshot 2024-04-11 at 18 11 05](https://github.com/prismicio/slice-machine/assets/19946868/03902863-4d07-4673-a49f-8253b4da9165)

Reply:
Good catch, it should be before UID so the group fields are on the same line.

From a more general view, should we even have UID as an option? We already add a UID field to repeatable custom/page types. If someone adds a single type, they shouldn't be using a UID since the type itself is the identifier.

Reply:
If you think that's worth discussing, should we bring it up to Côme?

Reply:
(I moved the UID field after the group fields in the meantime.)